datahub-web-react/src/app/entity/shared/components/styled/ExpandedActorGroup.tsx

```diff
@@ -4,7 +4,9 @@ import styled from 'styled-components';
 import { CorpGroup, CorpUser } from '../../../../../types.generated';
 import { ExpandedActor } from './ExpandedActor';
 
-const PopoverActors = styled.div``;
+const PopoverActors = styled.div`
+    width: 600px;
```

Comment:
should this be `max-width` and not `width` in case there are only a few actors?

Reply:
ooh great point